pldoc/static/sass/_lib.scss

```diff
@@ -13,3 +13,5 @@
 } @else {
     @import '../../../bower_components/bi-app-sass/bi-app/bi-app-ltr';
 }
+
+@import 'syntax';
```

Comment:
I was calling it in from a vendor file, but I've made some tweaks and I wanted to simplify the Sass/CSS folder structure, so importing it as a Sass partial made sense.